docs/spec/executable/tes-http-serverd.md

```diff
@@ -6,39 +6,28 @@ credentials for other processes to acquire.
 For information on TES, see
 <https://docs.aws.amazon.com/greengrass/v2/developerguide/interact-with-aws-services.html>.
 
-The following is the current daemon implementation flow:
+The following are the requirements for the `tes-http-serverd`:
 
 - [tes-http-serverd-1] The `tes-http-serverd` daemon will use the `tes` daemon
-  for acquiring credentials.
-- [tes-http-serverd-2] On connection, the `tes-http-serverd` daemon will
-  validate the provided token and establish an http connection with the server.
+  for acquiring TES credentials.
+- [tes-http-serverd-2] The `tes-http-serverd` daemon will validate client connections
+  with the provided `AWS_CONTAINER_AUTHORIZATION_TOKEN` (see [tes-http-serverd-ev-#]).
```

Comment:
Where can I find `tes-http-serverd-ev`?

Reply:
tes server does not get AWS_CONTAINER_AUTHORIZATION_TOKEN.
The client gets the token from its env and sends it in the request.